Codegen/src/Generators/LinearNodeSourceCodeMapper.cs

```diff
@@ -6,15 +6,14 @@
 using System.Linq;
 using System.Runtime.CompilerServices;
```

Comment:
This directive may be deleted too ?

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/pull/1648/commits/e087d9212c17eba494fe8cca891e34783e2e4570